project/layersAfterManifestImport.json

```diff
@@ -0,0 +1,38 @@
+[
```

Comment:
Generally this is out of scope here.  These are the resulting layers after importing https://tpen-project-examples.habesoftware.app/transcription-project/manifest.json